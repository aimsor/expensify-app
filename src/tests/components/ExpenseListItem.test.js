import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render ExpenseListItem with a single expense', () => {
  const wrapper = shallow(<ExpenseListItem expense={expenses[1]}/>);
  expect(toJSON(wrapper)).toMatchSnapshot();
});