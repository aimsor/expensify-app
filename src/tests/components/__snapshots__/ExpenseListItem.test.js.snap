// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with a single expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    $1,300.00
     - 
    Sat December 27th, 1969
  </p>
</div>
`;
